src/components/stepper/stepper.tsx

```diff
@@ -143,7 +143,8 @@ export class Stepper {
     event.stopPropagation();
   }
 
-  @Listen("calciteInternalStepperItemRegister") registerItem(event: CustomEvent): void {
+  @Listen("calciteInternalStepperItemRegister")
+  registerItem(event: CustomEvent): void {
```

Comment:
made this change to avoid JSDOC warning . unrelated to the issue.

Reply:
You mean linting warning?

Reply:
yup. linting warning.